app/src/main/java/com/fbada/projectone/domain/Car.kt

```diff
@@ -0,0 +1,350 @@
+/**
+ * Colours available as part of [Car] specification
+ */
+enum class AvailableColours { Red, Blue, Green, Yellow, Silver, White, Black }
+
+const val petrol = "Petrol"
+const val auto = "Automatic"
+const val manual = "Manual"
+const val big = "Big"
+const val leather = "Leather"
+const val lux = "luxury"
+/**
```

Comment:
Should be uppercase and placed at the very top of the file.

Reply:
renamed